### PR TITLE
Update readme to add examples

### PR DESCRIPTION
**Pull request by alikhajeh1**
Change 4th script to output unique values for an attribute key

### CODE REVIEW

**Comment by alikhajeh1** [comment]
@m-yosefpor can you please review this?

File: scripts/price_explorer/README.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+  ```sh
+  wget https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/index.json
+  ```
+
+## Usage
+
+* List all available AWS `service` names, or find a `service` name by searching for a keyword or acronym in the JSON file downloaded in the setup step. AWS use many acronyms so be sure to search for those too, e.g. "ES" returns "AmazonES" for ElasticSearch. The reason we do not query the pricing service graphQL API is that it only returns a limited number of responses, and as there are lots of products in AWS, our desired service name might not be included in the returned responses.
+
+  List all available AWS service names:
```
Comment (alikhajeh1) [inline scripts/price_explorer/README.md:25]: @sashman hopefully this should help find service names faster.

File: scripts/price_explorer/README.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    "MariaDB"
+    "MySQL (on-premise for Outpost)"
+    "MySQL"
+    "Oracle"
+    "PostgreSQL (on-premise for Outposts)"
+    "PostgreSQL"
+    "SQL Server"
+    ```
+
+* When writing integration tests, a `PriceHash` is used to match the price of a cost component so the tests continue to pass even if the actual price value changes. To find a `PriceHash`, browse to [https://pricing.infracost.io/graphql](https://pricing.infracost.io/graphql) and use the following query/variables:
```
Comment (alikhajeh1) [inline scripts/price_explorer/README.md:95]: @sashman does this help writing tests?

**Review by aliscott — APPROVED** [review]
LGTM

**Comment by alikhajeh1** [comment]
Going to merge so new contributors can use this, @m-yosefpor / @sashman plz leave any comments here so I can keep tweaking.

**Comment by sashman** [comment]
❤️  thanks very much @alikhajeh1 @aliscott!

**Pull request merged**